src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/media/index.test.js

```diff
@@ -0,0 +1,8 @@
+import convertMedia from '.';
+import { CPSMediaBlock, optimoVideoBlock } from './fixtures';
+
```

Comment:
Suspicious lack of unit tests here but I can't find all that much variation in the possible inputs - CPS video blocks all seem very consistent - welcome ideas on what more needs tested here!

Reply:
Should make sure we test with some older videos as well as newer ones as things behind the scenes will have changed over time.  Maybe pick a MAP beginning with each digit (1-5) to test?

Reply:
Yes, good point. We can only test against our fixed CPS assets in test for the time being, but we'll be sure to validate against older MAPs when Mozart have enabled the routing. I've created a generic issue here to capture the testing:https://github.com/bbc/simorgh/issues/4393